### PR TITLE
README.md overhaul + minor changes

### PR DESCRIPTION
**Pull request by Skerse**
# Pull Request

## Description

Complete readme overhaul. It is now up-to-date, more informational, and in my opinion, has a lot better formatting and cleaner code. Other than that, it remained mostly the same in terms of what there is.

Also did some minor changes to hypr configs, some of which i missed in my last pull request, and fit well with the readme overhaul. ~~Additionally in keybindings.conf, i removed the `$mainMod` variable as it was kind of useless in my opinion.~~

## Type of change

Please put an `x` in the boxes that apply:

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [x] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [ ] **Other** (provide details below)

## Checklist

Please put an `x` in the boxes that apply:

- [x] I have read the [CONTRIBUTING](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My commit message follows the [commit guidelines](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added a changelog entry.
- [ ] I have added necessary comments/documentation to my code.
- [ ] I have added tests to cover my changes.
- [x] I have tested my code locally and it works as expected.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Configs/.config/hypr/keybindings.conf
Comment (kRHYME7) [inline Configs/.config/hypr/keybindings.conf:1]: I think We should keep $mainMod. 

Not all keyboard might be mapped with SUPER or they don't use SUPER as their main Modifier

Comment (Skerse) [reply Configs/.config/hypr/keybindings.conf:1]: Fair, now that i think about that. I just thought it was kind of unneeded, but for anyone that wants to change it they should be able too easily. Will revert.

**Review by kRHYME7 — APPROVED** [review]
🤟

File: README.md [hunk 1]
```diff
@@ -1,262 +1,256 @@
...
-
-
-<https://user-images.githubusercontent.com/106020512/235429801-e8b8dae2-c1ad-4e23-9aa2-b1edb6cabe99.mp4>
-
-<p align="center">
-    <img align="center" width="49%" src="https://raw.githubusercontent.com/prasanthrangan/hyprdots/main/Source/assets/showcase_1.png" /> <img align="center" width="49%" src="https://raw.githubusercontent.com/prasanthrangan/hyprdots/main/Source/assets/showcase_2.png" />
-    <img align="center" width="49%" src="https://raw.githubusercontent.com/prasanthrangan/hyprdots/main/Source/assets/showcase_3.png" /> <img align="center" width="49%" src="https://raw.githubusercontent.com/prasanthrangan/hyprdots/main/Source/assets/showcase_4.png" />
-</p>
-
-https://github.com/prasanthrangan/hyprdots/assets/106020512/c995699a-299c-4a8e-99d4-71aa3d68eb93
```
Comment (Enigma1309) [inline README.md:20]: is there any need to remove showcase video? ( we should have updated it instead...)
i guess it looks good and will attract a wider audience.

apart from that it lgtm

**Review by Enigma1309 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Skerse) [reply README.md:20]: > is there any need to remove showcase video?

No, i didn't *need* to. I was definitely trying to make it cleaner and reduce the length of the readme. Also, the showcase videos are in the YouTube playlist linked at the bottom!

Comment (Enigma1309) [reply README.md:20]: okay let's see what @prasanthrangan has to say

**Comment by Skerse** [comment]
Marking as draft since i realized i missed something.

By the way @kRHYME7, should i add the discord link to the readme? I feel like it makes sense to add that now.

**Comment by kRHYME7** [comment]
I think it should be fine.

**Comment by Skerse** [comment]
It's ready now, made the fix (and added the discord) and put them back in 2 commits.

**Review by rubiin — APPROVED** [review]

**Comment by rubiin** [comment]
LGTM looks nice

**Review by prasanthrangan — APPROVED** [review]
looks good, thanks!

**Pull request merged**